files/zh-cn/web/api/eyedropper/index.md

```diff
@@ -0,0 +1,115 @@
+---
+title: EyeDropper
+slug: Web/API/EyeDropper
+---
+
+{{securecontext_header}}{{APIRef("EyeDropper API")}}{{SeeCompatTable}}
+
+**`EyeDropper`** 接口表示一个拾色器工具的实例，用户可以打开并使用它从屏幕上选择颜色。
+
+## 构造函数
+
+- {{DOMxRef("EyeDropper.EyeDropper", "EyeDropper()")}} {{Experimental_Inline}}
+  - : 返回一个新的 EyeDropper 实例。
+
+## 实例方法
+
+_`EyeDropper` 接口不继承任何方法_。
+
+- {{DOMxRef("EyeDropper.open()")}} {{Experimental_Inline}}
+  - : 返回一个 Promise，该 Promise 在解析时提供了一个对象，该对象可以访问所选颜色。
+
+## 示例
+
+### 打开拾色器工具并进行颜色采样
+
+以下示例演示如何打开拾色器工具，并等待用户选择屏幕上的像素，或按下 <kbd>Escape</kbd> 键取消拾色器模式。
+
+#### HTML
+
+```html
+<button id="start-button">Open the eyedropper</button> <span id="result"></span>
+```
+
+#### JavaScript
+
+```js
+document.getElementById("start-button").addEventListener("click", () => {
+  const resultElement = document.getElementById("result");
+
+  if (!window.EyeDropper) {
+    resultElement.textContent =
+      "Your browser does not support the EyeDropper API";
+    return;
+  }
+
+  const eyeDropper = new EyeDropper();
+
+  eyeDropper
+    .open()
+    .then((result) => {
+      resultElement.textContent = result.sRGBHex;
+      resultElement.style.backgroundColor = result.sRGBHex;
+    })
+    .catch((e) => {
+      resultElement.textContent = e;
+    });
+});
+```
+
+#### 结果
+
+{{EmbedLiveSample("Opening the eyedropper tool and sampling a color")}}
```

Comment:
Please see https://developer.mozilla.org/zh-CN/docs/MDN/Writing_guidelines/Page_structures/Live_samples#%E8%BF%90%E8%A1%8C%E5%AE%9E%E4%BE%8B%E7%B3%BB%E7%BB%9F%E5%A6%82%E4%BD%95%E5%B7%A5%E4%BD%9C to get the details

```suggestion
{{EmbedLiveSample("打开拾色器工具并进行颜色采样")}}
```